2018-10-Vancouver/agenda.md

```diff
@@ -0,0 +1,131 @@
+
+
+> __Vancouver Convention Centre West Building__
+1055 Canada Place
+Vancouver, British Columbia V6C 0C3
+Canada
+
+
+## Collaborator Summit Day One: October 12th, 2018
+
+1:00 PM - 5:00 PM
+
+### Agenda
+
+We have a big room and 2 breakout rooms available:
+
+* 1 big room for 300 people. (Main)
+* 2 rooms for 60 people each. (Breakout 1 and Breakout 2)
+
+_We are expecting 300 people to participate_.
+
+<br>
+
+
+| Time          | Place        | Topic                                         | Speakers/Champions                     | Issue                                      |
+|---------------|--------------|-----------------------------------------------|----------------------------------------|--------------------------------------------|
+| 1:00          | Main         | Intro, Sponsor Talk, Code of Conduct          | @mcollina @dshaw @chowdhurian          |                                            |
+| 1:15          | Main         | V8 Release Schedule and Bisection             | @hashseed                              | https://github.com/nodejs/summit/issues/100#issuecomment-409152001 |
+| 1:45          | Main         | The Embedder's POV                            | @codebytere                            | https://github.com/nodejs/summit/issues/100#issuecomment-409629177 |
+| 2:15          | Main         | Automation of Node.js core workflows          | @joyeecheung                           | https://github.com/nodejs/summit/issues/100#issuecomment-416011657 |
+| 2:45          |              | Break                                         |                                        | |
+| 3:15          | Main         | Session presentation, set up the Main room    | @mcollina @dshaw @chowdhurian          | |
+| 3:30          | Main         | Moderation                                    | @hackygolucky                          | |
+| 3:30          | Main         | Mentorship                                    | @Bamieh                                | |
+| 3:30          | Main         | LTS + release                                          |    @MylesBorins          | https://github.com/nodejs/Release/issues/358  |
+| 3:30          | Main         | Free                                          |                                        | |
+| 3:30          | Main         | Free                                          |                                        | |
+| 3:30          | Breakout 1   | Diagnostics                                   | @yunong                                | |
+| 3:30          | Breakout 2   | Promises                                      | @BridgeAR                              | |
+
+
+_If you want to speak and present a topic in the main room, feel free to reply to this issue with a title and abstract. If you want to propose a full session in a breakout room, add a link to an issue in this repo, similar to https://github.com/nodejs/summit/issues/86_.

```

Comment:
Can you change this from “reply to this issue” to “send a PR to update this document”?